.github/workflows/google.yml

```diff
@@ -0,0 +1,109 @@
+name: Build and Deploy to GKE
+
+on:
+  workflow_dispatch:
+
+env:
+  PROJECT_ID: cs3219-402414
+  GAR_LOCATION: asia-southeast1
+  REPOSITORY: peerprepimages
+  USER_IMAGE: user
+  QUESTIONS_IMAGE: questions
+  MATCHING_IMAGE: matching
+  COLLAB_IMAGE: collab
+  CLIENT_IMAGE: client
+
+jobs:
+  setup-build-publish-deploy:
```

Comment:
can I confirm if this deploys? If it doesnt then LGTM feel free to merge

Reply:
Theoretically, it should. In practice, there is no 100% way to be sure beside merging and running on this repo.

Reply:
I rather it not deploy, could you update it to not deploy? By deploy i mean update the pods haha

Reply:
Oh. Okay, I got the wrong thing. No, it doesn't update the pods at all. I deleted the deploy and it stops after pushing to artifact registry.